src/material-experimental/mdc-core/option/option.ts

```diff
@@ -37,6 +37,7 @@ import {MatOptgroup} from './optgroup';
     '[class.mat-mdc-option-active]': 'active',
     '[class.mdc-deprecated-list-item--disabled]': 'disabled',
     '[id]': 'id',
+    '[attr.aria-checked]': '_getAriaChecked()',
     '[attr.aria-selected]': '_getAriaSelected()',
```

Comment:
Doesn't the `aria-checked` conflict with `aria-selected`? I'd expect the screen reader to read out "Some option, checked, selected".